<launch>
  <arg name="rvizconfig" default="$(find struct_slam)/rviz/marker1.rviz" />

  <node name="points_and_lines" pkg="struct_slam" type="points_and_lines" />
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />
</launch>
